app/src/DesignSystem/Select.svelte

```diff
@@ -30,7 +31,7 @@
   }
 </style>
 
-<select {style} bind:value>
+<select data-cy={dataCy} {style} bind:value>
```

Comment:
Are all these `data-cy` attributes to make cypress work?

Reply:
Yes, these are test harnesses for cypress: https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements

Reply:
It seems odd to have this leak into our markup. Maybe a function of getting used to. Just wonder how leaky it is.

Reply:
I agree, it is a bit weird to add this stuff only for tests. But apparently it's a best practice.

We could potentially strip them out from a production build with some rollup magic if they're getting too much in the way.

Reply:
:black_heart: